native/src/routes/SearchModal.tsx

```diff
@@ -22,21 +24,27 @@ const Wrapper = styled.View`
   background-color: ${props => props.theme.colors.backgroundColor};
 `
 
+const ProgressSpinnerContainer = styled.View`
+  height: 50%;
```

Comment:
![image](https://github.com/user-attachments/assets/d63fd45b-402b-4e42-8e56-afd3c97c7fba)

It's not in the middle if that what  you are going for...

Reply:
No, I wanted it to show above the keyboard 😅